package.json

```diff
@@ -7,14 +7,18 @@
     "@cosmjs/cli": "^0.27.1"
   },
   "dependencies": {
-    "ipfs-car": "^0.6.1"
+    "@pinata/sdk": "^1.1.23",
+    "ipfs-car": "^0.6.1",
+    "ts-node": "^10.5.0",
+    "typescript": "^4.5.5"
   },
   "scripts": {
     "pack-images": "ipfs-car --pack images --output images.car",
     "pack-metadata": "ipfs-car --pack metadata --output metadata.car",
     "account": "cosmwasm-cli --init scripts/account.ts --code 'process.exit(0)'",
     "init": "cosmwasm-cli --init scripts/init.ts --code 'process.exit(0)'",
     "mint": "cosmwasm-cli --init scripts/mint.ts --code 'process.exit(0)'",
-    "load": "cosmwasm-cli --init scripts/load.ts"
+    "load": "cosmwasm-cli --init scripts/load.ts",
+    "upload": "ts-node scripts/ipfs-upload.ts"
```

Comment:
I would rename this to "pinata-upload" since other options may be added in the future like NFT.storage, Arweave, etc.